readme.md

```diff
@@ -255,6 +256,21 @@ Dim the color (emit a small amount of light).
 
 <img src="media/text-dimColor.jpg" width="138">
 
+#### inverse
+
+Type: `boolean`\
+Default: `false`
+
+Inverse background and foreground colors.
+
+```jsx
+<Text color="yellow" inverse>
+	Inversed Yellow
+</Text>
+```
+
+<img src="media/text-inverse.png" width="138">
```

Comment:
Can you use a jpg format instead?

Reply:
Done in https://github.com/vadimdemedes/ink/pull/347/commits/354d09a4317dbb61fdde1187b70d99c1997f650f